sycl-fusion/passes/kernel-fusion/SYCLKernelFusion.cpp

```diff
@@ -62,7 +62,7 @@ template <typename T> struct GetIntFromMD {
 static jit_compiler::Indices getIdxFromMD(const Metadata *MD) {
   constexpr unsigned IndicesBitWidth{64};
 
-  jit_compiler::Indices Res;
+  jit_compiler::Indices Res{0, 0, 0};
```

Comment:
Should we add a all-zero default constructor to `Indices`?

Reply:
Added, and then removed that again because it turns out that `Indices` can be completely immutable.

Reply:
... and added again 🙈; mutable `Indices` are useful in the following PRs.